lib/experimental/Widgets/ChartWidgetEmptyState/index.tsx

```diff
@@ -0,0 +1,90 @@
+import { Button, ButtonProps } from "@/components/Actions/Button"
+import { IconType } from "@/components/Utilities/Icon"
+import { cn } from "@/lib/utils"
+import { Card, CardContent, CardHeader, CardTitle } from "@/ui/card"
+import { forwardRef } from "react"
+import EmptyBarChart from "./Backgrounds/EmptyBarChart"
+import EmptyLineChart from "./Backgrounds/EmptyLineChart"
+
+type Type = "bar-chart" | "line-chart"
+
+export interface Props {
+  title: string
+  content: string
+  buttonLabel?: string
+  buttonIcon?: IconType
+  buttonAction?: () => void
+  type: Type
+}
+
+const ROOT_CLASSNAMES: Record<Type, string> = {
+  "line-chart": "border-f1-border-accent-alpha20",
+  "bar-chart": "border-f1-border-promoteAlpha30",
```

Comment:
is it correct? the other classnames follow different naming pattern

Reply:
Fixed it!